static/css/tocbot.css

```diff
@@ -0,0 +1,57 @@
+.toc {
```

Comment:
Why do we need this css file at the top level instead of inside the theme (we're already using a custom theme): 

`./themes/jaeger-docs/static/css/tocbot.css`

Reply:
Ah ok, I can edit the theme CSS, which I'd prefer, but I wasn't sure if this was ok given that I believe we usually override. Thanks for the tip. I'll update.